layouts/partials/common/footer.html

```diff
@@ -19,6 +19,7 @@ <h5>Resources</h5>
                     <li><a href="{{ "/get-almalinux" | relLangURL }}">{{ i18n "Downloads" }}</a></li>
                     <li><a href="{{ "/members" | relLangURL }}">{{ i18n "Membership" }}</a></li>
                     <li><a href="{{ "/elevate" | relLangURL }}">ELevate</a></li>
+                    <li><a href="/security.txt">security.txt file</a></li>
```

Comment:
It's a personal preference, but I think `security.txt file` looks weirder and less spiffy than just `security.txt`. This is 100% my opinion.

Reply:
Probably right, I'll change it.